flang/runtime/extensions.cpp

```diff
@@ -37,5 +80,17 @@ void FORTRAN_PROCEDURE_NAME(getarg)(
   (void)RTNAME(GetCommandArgument)(
       n, &value, nullptr, nullptr, __FILE__, __LINE__);
 }
+
+void FORTRAN_PROCEDURE_NAME(getlog)(std::int8_t *arg, std::int64_t length) {
+  std::array<char, LOGIN_NAME_MAX + 1> str = {};
+
+  int error = getlogin_r(str.data(), str.size());
```

Comment:
```suggestion
  int error{getlogin_r(str.data(), str.size())};
```

Reply:
Thanks a lot! I didn't know that these can be braced-initialized as well...